udata_croquemort/checker.py

```diff
@@ -24,13 +24,21 @@ def _format_response(self, response):
             check_date = response.get('updated')
             if check_date:
                 check_date = dateutil.parser.parse(response.get('updated'))
-            return {
+
+            result = {
                 'check:url': response.get('checked-url'),
                 'check:status': status,
                 'check:available': status and status >= 200 and status < 400,
                 'check:date': check_date,
             }
 
+            for header in [
+                'content-type', 'content-length', 'content-md5', 'charset'
+            ]:
+                result[f"check-headers:{header}"] = response.get(header)
```

Comment:
After reading https://github.com/opendatateam/udata/blob/25fa4ee05726e101539ed2d152b14ac842fdb6e7/udata/linkchecker/checker.py#L8, maybe we could settle on `check:headers:xxx`. This would allow this PR to work w/o modification on udata's side. WDYT?